src/h5web/vis-packs/core/line/MappedLineVis.tsx

```diff
@@ -66,6 +75,10 @@ function MappedLineVis(props: Props) {
   const auxiliaries = Object.values(useDatasetValues(auxDatasets));
   const [auxArrays, auxForDomain] = useMappedArrays(auxiliaries, ...hookArgs);
 
+  const title =
+    useDatasetValue(titleDataset) || valueLabel || valueDataset.name;
```

Comment:
Let's remove `valueLabel` from the props and use `getDatasetLabel(valueDataset)` as in `MappedHeatmapVis`

Reply:
There's a subtlety: if present, `valueLabel` is used as the ordinates label as well. This is the case with `NxSpectrumContainer` (but not `LineVisContainer`, which only displays the `title`).

Reply:
True but that does not prevent you from removing `valueLabel` from the props and instead define it as `valueLabel = getDatasetLabel(valueDataset)` in `MappedLineVis`.

Reply:
Left as is, as discussed.